content/pages/roadmap.html

```diff
@@ -16,61 +16,98 @@ <h4><i class="fa fa-map"></i> <b>Roadmap</b></h2>
+                <h2>Message Encoding</h2>
 
-                <p><a href="https://github.com/xu-hao/QueryArrow/">QueryArrow</a> has been developed to help alleviate this restriction and allow iRODS to store and query metadata in a variety of other database or indexing technologies.</p>
+                <p>iRODS implements its own message encoding via a library call PackStruct. PackStruct supports a binary format and an XML-like format. The only form of documentation for these encodings is the source code. The issue with using the source code as a reference for understanding PackStruct is that PackStruct's implementation is difficult to understand. Given iRODS' history, this is understandable. However, there are several encoding libraries available that provide features such as efficient binary encoding, versioning, and cross-language support. iRODS is at a point where it must switch to using a well maintained message encoding.</p>
 
-                <p>We plan to release the QueryArrow database plugin which will afford this abstraction of metadata.</p>
+                <p>The Consortium will be introducing a new plugin interface that gives developers a way to experiment with different message encoding libraries. Adding this interface also means the server and clients must learn how to negotiate which encoding to use. The first plugin implementation will be PackStruct.</p>
+
+                <h2>Cloud-Friendly</h2>
+
+                <div>
+                    <p>Cloud environments offer many benefits. One of the primary reasons for running applications in the cloud is scalability. While iRODS can be run in a cloud environment, scaling iRODS horizontally requires the same amount of work as if running iRODS on premise. Getting iRODS to the point where administrators can turn a dail and get more iRODS nodes will require some real thought.</p>
+
+                    <p>To improve the scalability situation, iRODS must satisfy the following requirements:</p>
+                    <ul>
+                        <li>Fast startup time</li>
+                        <li>Improved handling of host names</li>
+                        <li>Every server becomes a Provider</li>
+                    </ul>
+                </div>
+
+                <h2>Improved support for Archive files</h2>
+
+                <p>The facilities provided by iRODS to manipulate archive files needs to be modernized. The codebase contains several special code paths for dealing with archive files. This makes the implementation brittle and difficult to reason about. Users and administrators need tools with clear semantics. Therefore, we are planning on reimplementing support for archive files from the ground up.</p>
```

Comment:
"manipulate archive files" -> "manipulate archive files in various formats"

Reply:
done